scss/_global_functions.scss

```diff
@@ -62,14 +62,23 @@
 
 // Adds visual focus to elements on :focus-visible,
 // or :focus if the browser doesn't support the former
-@mixin vf-focus($color: $colors--theme--focus, $width: $bar-thickness, $has-validation: false) {
+@mixin vf-focus($light-color: $colors--light-theme--focus, $dark-color: $colors--dark-theme--focus, $width: $bar-thickness, $has-validation: false) {
+  .is-light &,
+  .is-paper & {
+    --vf-focus-outline-color: #{$light-color};
+  }
```

Comment:
What happens if `.is-light`, `.is-paper`, and `.is-dark` are all not present? I don't see `vf-focus-outline-color` defined elsewhere, maybe this should default to light theme if none of these classes are found.